org.gnome.chess.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.gnome.chess",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "gnome-chess",
+    "copy-icon": true,
+    "rename-icon": "gnome-chess",
+    "rename-desktop-file": "gnome-chess.desktop",
+    "rename-appdata-file": "gnome-chess.appdata.xml",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* dconf */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        /* Documents directory access for storing saved games */
+        "--filesystem=xdg-documents"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup": ["/share/gnuchess", "/share/info", "/share/man"],
+    "modules": [
+        {
+            "name": "gnuchess",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ftpmirror.gnu.org/chess/gnuchess-6.2.3.tar.gz",
+                    "sha256": "78999176b2f2b5e0325bcc69749b7b2cefb7b1ef4f02d101fa77ae24a1b31b82"
+                }
+            ]
+        },
+        {
+            "name": "gdk-pixbuf2",
+            "buildsystem": "meson",
+            "cleanup-platform": [ "/bin"],
+            "sources": [
+                 {
+                     "type": "archive",
+                     "url": "https://download.gnome.org/sources/gdk-pixbuf/2.37/gdk-pixbuf-2.37.0.tar.xz",
+                     "sha256": "81508700f500eca3f21c4cfd0a3486459a373a65e07457a9a4f3bf7d08109c11"
+                 }
+            ]
+        },
+        {
+            "name": "librsvg",
+            "config-opts": ["--disable-static", "--enable-vala"],
+            "ensure-writable": ["/lib/gdk-pixbuf-2.0/*/loaders.cache"],
```

Comment:
Most likely not needed due to different prefix.

Reply:
If I remove this, it will cause this error
```
/bin/sh: line 2: /app/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache: Read-only file system
make[4]: *** [Makefile:890: install-data-hook] Error 1
make[4]: Leaving directory '/run/build/librsvg/gdk-pixbuf-loader'
make[3]: *** [Makefile:806: install-data-am] Error 2
make[3]: Leaving directory '/run/build/librsvg/gdk-pixbuf-loader'
make[2]: *** [Makefile:752: install-am] Error 2
make[2]: Leaving directory '/run/build/librsvg/gdk-pixbuf-loader'
make[1]: *** [Makefile:1725: install-recursive] Error 1
make[1]: Leaving directory '/run/build/librsvg'
make: *** [Makefile:2031: install] Error 2
Error: module librsvg: Child process exited with code 2
```

Reply:
:+1: